Plugins/UnityServerPlugin/StreamingUnityServerPlugin.cpp

```diff
@@ -155,6 +155,11 @@ struct UnityServerPeerObserver : public PeerConnectionClientObserver,
 		}
 	}
 
+	virtual void OnHeartbeat(int heartbeat_status) override
+	{
+		// TODO(bengreenier): wire up to unity
```

Comment:
Can you create an issue for this?

Reply:
see https://github.com/CatalystCode/3DStreamingToolkit/issues/259